tests/rateLimit.tests.js

```diff
@@ -14,6 +14,7 @@ describe('JwksClient (cache)', () => {
   describe('#getSigningKeys', () => {
     it('should prevent too many requests', (done) => {
       const client = new JwksClient({
+        cache: false,
```

Comment:
Previously the cache was off by default for this test, turning off the cache to not change functionality in this test